pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: grpc
 description: Dart implementation of gRPC.
-version: 0.2.1
+version: 0.3.0
```

Comment:
Missing changelog entry to go with this version bump.

Reply:
And I was doing so well until now. Coming up!